wiki/Beginner/en.md

```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
--   [Hatsune Miku - Do-re-mi-fa Rondo](http://osu.ppy.sh/b/163457) on Easy (1.28 Stars)
+-   [Mutsuhiko Izumi - Russian Snowy Dance](https://osu.ppy.sh/b/124348) on Easy (1.21 Stars)
+-   [goreshit - o'er the flood](https://osu.ppy.sh/b/177139) on Easy (1.22 Stars)
+-   [Hatsune Miku - Electric Love (Short Ver.)](https://osu.ppy.sh/b/141749) on Easy. (1.25 Stars)
+-   [DJ YOSHITAKA feat. Hoshino Kanako - MAX LOVE](https://osu.ppy.sh/b/186601) on Easy (1.28 Stars)
+-   [Akcent feat. Edward Maya - That's My Name](https://osu.ppy.sh/b/132838) on Easy (1.28 Stars)
+-   [Hatsune Miku - Do-re-mi-fa Rondo](https://osu.ppy.sh/b/163457) on Easy (1.28 Stars)
 
-Challenge
----------
+### Challenge
 
 Want to push yourself and take on a challenge? Try these!
 
--   [DJ Fresh - Gold Dust](http://osu.ppy.sh/b/96042) on Easy (1.32 Stars)
--   [Kyary Pamyu Pamyu - Ninjya Re Bang Bang](http://osu.ppy.sh/b/230174) on Easy (1.36 Stars)
--   [Carly Rae Jepsen - Call Me Maybe (Nightcore Mix)](http://osu.ppy.sh/b/172736) on Easy (1.36 Stars)
--   [Kicks N Licks Feat. Nicole Millar - Own The Night (MitiS DnB Remix)](http://osu.ppy.sh/b/368307) on Easy (1.45 Stars)
--   [Gentle Stick X M2U - Hades in the Heaven](http://osu.ppy.sh/b/318994) on Easy (1.46 Stars)
+-   [DJ Fresh - Gold Dust](https://osu.ppy.sh/b/96042) on Easy (1.32 Stars)
+-   [Kyary Pamyu Pamyu - Ninjya Re Bang Bang](https://osu.ppy.sh/b/230174) on Easy (1.36 Stars)
+-   [Carly Rae Jepsen - Call Me Maybe (Nightcore Mix)](https://osu.ppy.sh/b/172736) on Easy (1.36 Stars)
+-   [Kicks N Licks Feat. Nicole Millar - Own The Night (MitiS DnB Remix)](https://osu.ppy.sh/b/368307) on Easy (1.45 Stars)
+-   [Gentle Stick X M2U - Hades in the Heaven](https://osu.ppy.sh/b/318994) on Easy (1.46 Stars)
 
-Going Beyond
-============
+## Going Beyond
 
-Once you have exhausted the difficulty of the beatmaps listed here, don't stop there. Keep on going, and find some maps of a higher difficulty rating. Try even searching for beatmaps based on [difficulty suggestion](http://osu.ppy.sh/p/beatmaplist?s=3&o=-1&m=0&r=0&g=0&la=0&ra), as it will help you get an idea of where you need to go next. Don't be afraid to ask other users too about how you can improve, most often they will be nice and share you a pretty fun map to play that they also enjoyed, where their osu! world adventure begun. Most importantly, have fun!
+Once you have exhausted the difficulty of the beatmaps listed here, don't stop there. Keep on going, and find some maps of a higher difficulty rating. Try even searching for beatmaps based on [difficulty suggestion](https://osu.ppy.sh/p/beatmaplist?s=3&o=-1&m=0&r=0&g=0&la=0&ra), as it will help you get an idea of where you need to go next. Don't be afraid to ask other users too about how you can improve, most often they will be nice and share you a pretty fun map to play that they also enjoyed, where their *osu!* world adventure begun. Most importantly, have fun!
```

Comment:
fix this sentence
```
Don't be afraid to ask other users about improving, most often they will be nice and share you a pretty fun map to play that they also enjoyed, where their *osu!* world adventure begun. Lastly, but most importantly, have fun!
```
(and this is why I break all sentences down individually :/)